src/commands/monitors/run.rs

```diff
@@ -158,6 +193,18 @@ fn token_execute(
     (success, code, None)
 }
 
+fn parse_monitor_config_args(matches: &ArgMatches) -> Result<Option<MonitorConfig>> {
+    Ok(match matches.get_one::<String>("schedule") {
+        Some(schedule) => Some(MonitorConfig {
+            schedule: MonitorSchedule::from_crontab(schedule)?,
+            checkin_margin: matches.get_one("checkin_margin").copied(),
+            max_runtime: matches.get_one("max_runtime").copied(),
+            timezone: matches.get_one("timezone").cloned(),
+        }),
+        None => None,
+    })
```

Comment:
I would suggest splitting this up a bit.
```suggestion
    let Some(schedule) = matches.get_one::<String>("schedule") else {
        return Ok(None);
    };
    let schedule = MonitorSchedule::from_crontab(schedule)?;
    Ok(Some(MonitorConfig {
        schedule,
        checkin_margin: matches.get_one("checkin_margin").copied(),
        max_runtime: matches.get_one("max_runtime").copied(),
        timezone: matches.get_one("timezone").cloned(),
    }))
```